code/modules/antagonists/nukeop/nukeop.dm

```diff
@@ -26,8 +26,9 @@
 	if(!ishuman(owner.current))
 		return
 	var/mob/living/carbon/human/H = owner.current
-
-	H.set_species(/datum/species/human) //Plasamen burn up otherwise, and lizards are vulnerable to asimov AIs
+	
+	if(isplasmaman(H))
```

Comment:
Mmm, I don't really like how this uses a check in the antagonist equipping step.

Is there some way we can use some hook in the `/datum/species` datums, or create one?

Job code already has a [hook ](https://github.com/tgstation/tgstation/blob/master/code/modules/mob/living/carbon/human/species_types/plasmamen.dm#L81)in /datum/species.

Reply:
ain't there some variable or something for dangerous species?